src/test/regress/sql/with_modifying.sql

```diff
@@ -221,13 +221,15 @@ SELECT * FROM modify_table ORDER BY id, val;
 SELECT * FROM anchor_table ORDER BY id;
 
 INSERT INTO modify_table VALUES (11, 1), (12, 2), (13, 3);
-WITH select_data AS (
+
+-- This test will be enabled after recursive planner is enabled for modify query.
+/*WITH select_data AS (
```

Comment:
Maybe just leave the test as is, but with an output change. That'll make sure that we don't forget to re-enable it in the recursive planning PR.